docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  pg:
+    image: postgres:10.0-alpine

```

Comment:
We use sonatanfv/son-postgres as postgres image

Reply:
ok